tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml

```diff
@@ -110,6 +169,40 @@ jobs:
      copy $(Build.BinariesDirectory)\nuget-artifact\*.nupkg $(Build.ArtifactStagingDirectory)
     displayName: 'Bundle Native NuGet and other binaries'
 
+  - script: |
+     pushd $(Build.BinariesDirectory)\nuget-artifact-dml
+     dir
+     powershell -Command "Invoke-WebRequest http://stahlworks.com/dev/unzip.exe -OutFile unzip.exe"
+     powershell -Command "Invoke-WebRequest http://stahlworks.com/dev/zip.exe -OutFile zip.exe"
+     set PATH=%CD%;%PATH%
+     FOR /R %%i IN (*.nupkg) do ( 
+         rename %%~ni.nupkg %%~ni.zip
+         unzip %%~ni.zip -d %%~ni
+         del /Q %%~ni.zip
+
+         unzip win-dml-x86.zip -d win-x86
+         mkdir %%~ni\runtimes\win-x86
+         mkdir %%~ni\runtimes\win-x86\native
+
+         move win-x86\runtimes\win-x86\native\onnxruntime.dll %%~ni\runtimes\win-x86\native\onnxruntime.dll
+         move win-x86\runtimes\win-x86\native\onnxruntime.lib %%~ni\runtimes\win-x86\native\onnxruntime.lib
+         move win-x86\runtimes\win-x86\native\onnxruntime.pdb %%~ni\runtimes\win-x86\native\onnxruntime.pdb
+
+         move win-x86\runtimes\win-x86\native\windows.ai.machinelearning.dll %%~ni\runtimes\win-x86\native\windows.ai.machinelearning.dll
+         move win-x86\runtimes\win-x86\native\windows.ai.machinelearning.lib %%~ni\runtimes\win-x86\native\windows.ai.machinelearning.lib
+         move win-x86\runtimes\win-x86\native\windows.ai.machinelearning.pdb %%~ni\runtimes\win-x86\native\windows.ai.machinelearning.pdb
+
+         move win-x86\runtimes\win-x86\native\directml.dll %%~ni\runtimes\win-x86\native\directml.dll
```

Comment:
>dll [](start = 98, length = 3)

No pdb for DML?

Reply:
It wasn't part of the package earlier. Do we want to enable debugging dml in the public release?

Reply:
added